spec/lib/batch_load/import/dwca_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+require 'batch_load/import/collection_objects'
+
+describe BatchLoad::Import::DWCA, type: :model do
+  # iconv -f iso-8859-15 -t UTF-8  spec/files/batch/dwca/PSUC3-Test.txt >spec/files/batch/dwca/PSUC3-Test.utf8.txt
```

Comment:
Encapsulated in an explanatory context, but I think it is where it belongs, given that it is used to generate the test file for the spec.